packages/app-store/zapier/api/subscriptions/deleteSubscription.ts

```diff
@@ -27,13 +27,24 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
   const webhook = await prisma.webhook.findFirst({
     where: {
       id,
+      AND: [{ userId: validKey.userId }, { teamId: validKey.teamId }],
```

Comment:
I think this means that only users with a team and valid should be able to delete webhooks? Or does it works for users without a team (individuals)?